doc/build/conf.py

```diff
@@ -467,3 +469,20 @@
 
 # Allow duplicate toc entries.
 # epub_tocdup = True
+
+# Configuration for sphinxcontrib.spelling
+spelling_lang = "en_US"
+tokenizer_lang = "en_US"
+spelling_verbose = False
+spelling_ignore_pypi_package_names = True
+spelling_filters = [
+    "enchant.tokenize.URLFilter",
+]
+spelling_show_suggestions = True
+spelling_suggestion_limit = 2
+# All of the below are true by default
+# spelling_ignore_wiki_words = True
+# spelling_ignore_acronyms = True
+# spelling_ignore_python_builtins = True
+# spelling_ignore_importable_modules = True
+# spelling_ignore_contributor_names = True
```

Comment:
is there a configuration that indicates the location of the wordlist file?

Reply:
Yes there's `spelling_word_list_filename`, I just included the word filter related options. Do you want it to be explicit?

Full options here https://sphinxcontrib-spelling.readthedocs.io/en/latest/customize.html

Reply:
I think it would make sense to make it explicit yes, so searching the file name would make it obvious where it's used

Reply:
also a comment with the link to the option list is always handy :)